starwars/src/components/cardData.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Card from './card';
+ function CardData(props){
+    return(
+        <div className='card-container'>
+            {/* Mapping over CharacterData that holds starwarsChars and setting it to a new array
+            called charInfo*/}
+            {props.characterData.map(charInfo => 
+            <Card 
+            charInfo={charInfo} 
```

Comment:
Nice job! I would probably either pass just the charInfo object, or break it out into separate properties, but not both